prov/efa/src/efa_msg.c

```diff
@@ -169,7 +178,9 @@ static ssize_t efa_post_recv(struct efa_ep *ep, const struct fi_msg *msg, uint64
 	if (flags & FI_MORE)
 		return 0;
 
-	err = ibv_post_recv(qp->ibv_qp, ep->recv_more_wr_head.next, &bad_wr);
+	err = -ibv_post_recv(qp->ibv_qp, ep->recv_more_wr_head.next, &bad_wr);
```

Comment:
why all the negate math?

Reply:
This math is trying to convert an ibv return code (positive) to Libfabric return code (negative).

But I understand doing math on return code is counter-intuitive, so in new revision I introduced an inline function `efa_ibv_retcode_to_fi_retcode`.